src/coreclr/debug/daccess/daccess.cpp

```diff
@@ -6716,12 +6660,6 @@ ClrDataAccess::GetMetaDataFromHost(PEAssembly* pPEAssembly,
         {
             goto ErrExit;
         }
-        // Transform NGEN image name into IL Image name
-        if (!GetILImageNameFromNgenImage(ilExtension, uniPath, ARRAY_SIZE(uniPath)))
```

Comment:
I'm don't know if this transform is needed or not anymore.  @noahfalk?

Reply:
I am not aware that we use .ni.dll as a naming convention any longer. I don't think this is needed.

Reply:
cc @ gbalykov, I noticed (in #55663) that Tizen team uses .ni.dll suffix.

Reply:
Thanks for mentioning, we'll need to investigate how this affects us. cc @alpencolt

Reply:
@gbalykov For now, I am going to consider this impact as a possibility rather than a blocker. If any Tizen stakeholder finds an issue with this change, we will of course bring back the specific support. Additionally it will help us to update the comment.

Reply:
There are different places in runtime, which check and consider `ni.dll` extension, like `APP_NI_PATHS` for instance. Do you have any plans to fully remove `ni.dll` extension support from runtime?

Reply:
@am11 thanks! I've missed this change somehow